Source/WebCore/html/track/VTTCue.cpp

```diff
@@ -148,6 +148,29 @@ VTTCueBox::VTTCueBox(Document& document, VTTCue& cue)
 {
 }
 
+void VTTCueBox::applyCSSPropertiesWithRegion()
+{
+    auto textTrackCue = getCue();
+    ASSERT(!textTrackCue || is<VTTCue>(textTrackCue));
+    if (!textTrackCue || !is<VTTCue>(textTrackCue))
+        return;
+
+    Ref cue = downcast<VTTCue>(*textTrackCue);
+
+    // the 'left' property must be set to left
+    std::visit(WTF::makeVisitor([&] (double left) {
```

Comment:
Nit: may be a little simpler to use WTF::switchOn rather than std::visit + WTF::makeVisitor.